package.json

```diff
@@ -4,14 +4,14 @@
   "description": "Decoder and encoder for the Ethereum ABI",
   "main": "index.js",
   "dependencies": {
-    "bn.js": "^4.10.0",
-    "ethereumjs-util": "^5.0.0"
+    "bn.js": "4.11.8",
+    "ethereumjs-util": "6.0.0"
   },
   "devDependencies": {
-    "coveralls": "^2.11.4",
-    "istanbul": "^0.4.1",
-    "mocha": "^2.1.0",
-    "standard": "^5.4.1"
+    "coveralls": "3.0.3",
+    "istanbul": "0.4.5",
+    "mocha": "6.0.2",
+    "standard": "12.0.1"
```

Comment:
We use semantic versioning for our libraries, can you please keep the version ranges here? I know there are good reasons for alternative approaches, I nevertheless wouldn't want to introduce such things on the sideline. I you want to discuss around things like that please use our organization repository.